src/exasolvs/queryrenderer/AggregateFunctionAppender.lua

```diff
@@ -57,13 +60,57 @@ function AggregateFunctionAppender:_append_function_argument_list(arguments)
     self:_append(")")
 end
 
+function AggregateFunctionAppender:_append_distinct_function(f)
+    self:_append(string.upper(f.name))
+    local distinct = f.distinct or false
+    self:_append_function_argument_list(distinct, f.arguments)
+end
+
 function AggregateFunctionAppender:_append_simple_function(f)
+    assert(not f.distinct)
```

Comment:
Maybe add a message in case the assertion fails

Reply:
Fixed. Also added unit test.